.github/ISSUE_TEMPLATE/other.yml

```
@@ -0,0 +1,21 @@
+name: Other
+description: Use this for any other issues. PLEASE, do not create blank issues
```

```suggestion
description: Use this for any other issues. PLEASE do not create blank issues
```
Comma here is extraneous.